testsuite/test_ta_workflow.py

```diff
@@ -0,0 +1,776 @@
+from datetime import datetime, timezone
+
+import pytest
+
+from django.contrib.admin.sites import AdminSite
+
+from interface.models import Course, Submission
+from interface.admin import CourseAdmin
+
+
+class MockObject(object):
+    pass
+
+
+@pytest.mark.django_db
+def test_ta_add_new_ta(client, STC, base_db_setup):
+    '''
+        Create one teaching assistant, 5 courses and 3 assignments for
+        each course.
+
+        The TA is added to courses 1, 3, 5 and should see only those
+        assignments
+    '''
+    (super_user, ta, user, course, _) = base_db_setup
+
+    client.login(username=user.username, password='pw')
+    response = client.get('/admin/')
+    STC.assertRedirects(response, '/admin/login/?next=/admin/')
+
+    request = MockObject()
+    request.user = super_user
+
+    form = MockObject()
+    form.changed_data = ["teaching_assistants"]
+    form.cleaned_data = {"teaching_assistants": [user, ta]}
+
+    test_course_admin = CourseAdmin(model=Course, admin_site=AdminSite())
+    test_course_admin.save_model(
+                obj=course,
+                request=request,
+                form=form,
+                change=True)
```

Comment:
The parenthesis should be on the next line and `True` should be followed by a final comma.